docs/OV_Runtime_UG/migration_ov_2_0/deployment_migration.md

```diff
@@ -1,43 +1,43 @@
 # Installation & Deployment {#openvino_2_0_deployment}
 
-"Easy to use" is one of the main concepts for OpenVINO™ API 2.0. It includes not only simplifying the migration from frameworks to OpenVINO, but also how OpenVINO is organized, how the development tools are used, and how to develop and deploy OpenVINO-based applications.
+"Easy to use" is one of the main concepts for OpenVINO™ API 2.0. It is about simplifying migration from different frameworks to OpenVINO, but also touches on how OpenVINO is organized, how its development tools are used, and how OpenVINO-based applications are developed and deployed.
 
-To accomplish that, we have made some changes on the installation and deployment of OpenVINO in the 2022.1 release. This guide will walk you through them.
+To accomplish that, we made some changes to the installation and deployment processes of OpenVINO in the 2022.1 release. This guide will walk you through them.
 
-## Installer Package Contains OpenVINO™ Runtime Only
+## The Installer Package Contains OpenVINO™ Runtime Only
 
-Starting from OpenVINO 2022.1, Model Optimizer, Post-Training Optimization tool and Python-based Development tools such as Open Model Zoo tools are distributed via [PyPI](https://pypi.org/project/openvino-dev/) only, and are no longer included in the OpenVINO installer package. This change has several benefits as it:
+Starting from OpenVINO 2022.1, Model Optimizer, Post-Training Optimization tool, and Python-based Development tools, such as those of Open Model Zoo, are distributed via [PyPI](https://pypi.org/project/openvino-dev/) only, and are no longer included in the OpenVINO installer package. This change has several benefits, as it:
```

Comment:
Can we revert it back? The first three commas should not be added, and the last one is not necessary.